common/samd21g18a_firmware_partition.ld

```diff
@@ -50,7 +50,7 @@ SEARCH_DIR(.)
 MEMORY
 {
   rom    (rx)  : ORIGIN = 0x00001000, LENGTH = 0x0003efff
-  ram    (rwx) : ORIGIN = 0x20000000, LENGTH = 0x00001000
+  ram    (rwx) : ORIGIN = 0x20000000, LENGTH = 0x00008000
```

Comment:
Was this the source of any bugs? Why did you decide to make it 0x8000 instead of 0x1000?